### PR TITLE
Todo app: Add, Delete and Edit list 

### PR DESCRIPTION
**Pull request by maxthestranger**
- added addtask func to add tasks to the list
- added edit and remove function for the same
- updated the list index and added localStorage support

### CODE REVIEW

File: src/tasks.js [hunk 1]
```diff
@@ -0,0 +1,106 @@
+const input = document.querySelector('.input');
```
Comment (gemmen29) [inline src/tasks.js:1]: - [ ] Kindly for `clear all completed` task implement the same behavior in the requirement [video](https://www.youtube.com/watch?v=AcUd-_Yjjqg), not just clear the description of the task

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi @maxthestranger, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/tasks.js (hunk 1 above)
Comment (maxthestranger) [reply src/tasks.js:1]: Hi @gemmen29 the task for this day was just to implement add, edit and delete... tomorrow is when I am supposed to implement checkbox functionality.

Comment (gemmen29) [reply src/tasks.js:1]: The add and remove functionalities that should be implemented in this project, Is the `interactive list` project tomorrow or yesterday?

Comment (maxthestranger) [reply src/tasks.js:1]: tomorrow

Comment (maxthestranger) [reply src/tasks.js:1]: actually the ```interactive list``` is on Thursday ~ tomorrow is ```add remove and edit``` which is what I have done today and then today is ```building the list UI structure``` wich I did yesterday - so today you were to check for ```add remove edit```. thanks

Comment (gemmen29) [reply src/tasks.js:1]: Ok, I will contact the support because in our time when we do that project the `interactive list` is before the `add and remove`

Comment (maxthestranger) [reply src/tasks.js:1]: Screenshot

may they changed it, but it's okay

Comment (gemmen29) [reply src/tasks.js:1]: I called the support and wait for a response.

Comment (gemmen29) [reply src/tasks.js:1]: Sorry for that misunderstanding and your project will be approved manually.

**Pull request merged**